src/lib/SnackBar/index.ts

```diff
@@ -8,12 +8,9 @@ export interface SnackOptions {
 function createSnack(message: string, options: SnackOptions): [Element, Promise<string>] {
   const {
     timeout = 0,
-    actions = [],
+    actions = ['dismiss'],
```

Comment:
Snuck this in. Realised it doesn't make sense to have a snack that doesn't have a dismiss button.